config/manifests/bases/kaoto-operator.clusterserviceversion.yaml

```diff
@@ -13,27 +13,27 @@ spec:
-    mediatype: image/png
+  - base64data: ""
```

Comment:
Is the icon a TODO?

Reply:
I think I have removed it as part of some refactoring, will add it back